deps.bzl

```diff
@@ -5,9 +5,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file"
 # When changing this version, a new release of podman-static may be needed.
 # See dockerfiles/executor_image/README.md for instructions.
 # The checksums below will also need to be updated.
-PODMAN_VERSION = "v4.9.4"
-PODMAN_STATIC_SHA256_AMD64 = "ac14152d2ef5cb25abbd615893cd56355f951f075a7ae663f0b2c1bc5d3fb77e"
-PODMAN_STATIC_SHA256_ARM64 = "72ab4a9c6ae9d5d00200070cb43b0921117898925a284bfacbfb3873aa82e598"
+PODMAN_VERSION = "v5.1.1-1"
+PODMAN_STATIC_SHA256_AMD64 = "a172ed5383234e6b44a0de8d908bb0aa1baff8beeb56214898c3a7930a3c903d"
+PODMAN_STATIC_SHA256_ARM64 = "3f932655d812b26525063f1b13a0fbab918ea536aee3311e69805c075460ad5f"
```

Comment:
nit, split the podman upgrade into a separate PR? since it's a major version upgrade it's more risky and would be nice to have as a separate commit that we can bisect.